### PR TITLE
fix: update fetching desktop release endpoint

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/session/apis/snode_api/getNetworkTime.ts [hunk 1]
```diff
@@ -70,9 +70,16 @@ function getNowWithNetworkOffset() {
   return Date.now() - GetNetworkTime.getLatestTimestampOffset();
 }
 
+function getNowWithNetworkOffsetSeconds() {
```
Comment (yougotwill) [inline ts/session/apis/snode_api/getNetworkTime.ts:73]: Did this get removed at some point? I felt like this was always here? It's on my onboarding branch

File: ts/session/fetch_latest_release/index.ts [hunk 2]
```diff
@@ -35,7 +29,7 @@ async function fetchReleaseFromFSAndUpdateMain() {
       return;
     }
 
-    const justFetched = await getLatestReleaseFromFileServer();
+    const justFetched = await getLatestReleaseFromFileServer(userEd25519SecretKey);
     window.log.info('[updater] fetched latest release from fileserver: ', justFetched);
```
Comment (yougotwill) [inline ts/session/fetch_latest_release/index.ts:33]: Maybe log the key since it's blinded? I assume that is not a data leak.

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

File: ts/session/apis/snode_api/getNetworkTime.ts (hunk 1 above)
Comment (Bilb) [reply ts/session/apis/snode_api/getNetworkTime.ts:73]: probably just present on unstable, but not clearnet yet? I didn't find it anywhere

File: ts/session/fetch_latest_release/index.ts (hunk 2 above)
Comment (yougotwill) [reply ts/session/fetch_latest_release/index.ts:33]: My mistake resolved

**Review by yougotwill — APPROVED** [review]
LGTM 🚢

**Pull request merged**